### ISSUE TITLE
Upgrade discovery Error

### ISSUE THREAD

**Issue by long-lazuli** [will-stone/browserosaurus#145]
On my machine, I white-list some applications to access network.
So browserosaurus has, per default, no access to network.

I constantly have SSL network error from the App.
Can you consider a way to toggle auto-update ?

---

Code would beneficiate from silencing the error and avoiding this dialog, of course.
I think it's good too that the end-user can control which app does what.

Here is a screenshot of the dialog:

<img width="497" alt="image" src="https://user-images.githubusercontent.com/1178625/67798773-75c54380-fa84-11e9-95ff-b2214e0a1806.png">

If I let the dialog open and do something else.
Next time the network check happens, this occurs :
<img width="546" alt="image" src="https://user-images.githubusercontent.com/1178625/67800253-4237e880-fa87-11e9-807b-f9e9a3c8044f.png">

---

**Comment by will-stone** [comment]
Hi. Yes I totally agree: I hate the Electron autoUpdater. Well, "hate" is strong word, it does after all do a lot for me. However, I don't know why it insists on having to download the new version before letting the user know. I've recently pulled out the code from the `update-electron-app` package so that I can notify the user (by changing the colour of the menubar icon) without the obnoxious dialog. I will be looking into a way to stop the auto update check/download to make this process even smoother.

**Comment by rushiagr** [comment]
I am affected by this too. After the recent update, whenever there is a loss of network connectivity, similar dialogues show up, which is irritating. Never had this problem before, on the older version.

**Comment by long-lazuli** [comment]
Maybe we could start by hugely augmenting the delay between checking ?

I'd be unable to try (or make a PR) before few weeks but I did come across this line which might be useful.
Replacing TEN_MINS by ONE_FULL_DAY seems fairly enough for this project, to me.

https://github.com/will-stone/browserosaurus/blob/365e5767402b027e4ef896bcbeaa12adeeccdf7f/src/main/main.ts#L254

**Comment by will-stone** [comment]
Thanks @long-lazuli for the suggestions. I've just seen this error too.

When I used the `update-electron-app` package, this never used to happen, which suggests they solved it. I removed all the unused event listeners, so I'm guessing if I put this back in, the problem will go away:

```js
autoUpdater.on('error', err => {
  console.log('updater error')
  console.log(err)
})
```

I'll be releasing v7 soon and will put the fix in that. If that starts to look further away than today or tomorrow, I'll roll back and release as a fix on v6.
